### PR TITLE
Add New Featured Artist: LeaF newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

File: news/2020-03-18-new-featured-artist-leaf.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+![](https://assets.ppy.sh/artists/73/header.jpg)
+
+Those with particularly calamitous fortunes are likely well acquainted with the wonder that is **LeaF** and his genre-defying rhythmic mastery. From tracks like *Calamity Fortune*, *MEPHISTO*, *Wizdomiot*, *I*, *Evanescent* and more, we've seen a LOT of **LeaF** over the course of osu!'s history, with many of the beatmaps charted to his work serving as cultural icons in their own right.
+
+We've lined up **30** tracks for you to test your mapping mettle with, in a variety of short and extended versions to boot.
+
+In case you're not in the know, check out a sample of some of his work below:
+
+### Aleph-0
```
Comment (cl8n) [inline news/2020-03-18-new-featured-artist-leaf.md]: there are popular maps for this one

https://osu.ppy.sh/beatmapsets/791900
https://osu.ppy.sh/beatmapsets/518951
https://osu.ppy.sh/beatmapsets/884669

File: news/2020-03-18-new-featured-artist-leaf.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/73/previews/1980.mp3" type="audio/mpeg">
+</audio>
+
+### Calamity Fortune
+
+Check out this [extremely popular classic beatmap](https://osu.ppy.sh/beatmapsets/65855#osu/192515) hosted by [Flower](https://osu.ppy.sh/users/1033017)!
```
Comment (cl8n) [inline news/2020-03-18-new-featured-artist-leaf.md]: should link just the set, not a diff (for all of these)

File: news/2020-03-18-new-featured-artist-leaf.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/73/previews/2005.mp3" type="audio/mpeg">
+</audio>
+
+------
+
+All of the above plus a whole lot more are available right now over at [**LeaF**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/73), so head on over and sink your teeth into a collection of tracks that thrived during the golden age of osu! mapping, all 100% free and just waiting for your fresh new take.
+
+Tweeters @TheEggowo, @DrLou15, @iSpecYou and @sucuhaamane all take home the prize of correctly nabbing this week's hint. Congratulations! For those wondering, 'verzweiflung' briefly features as accent text in the video for *Calamity Fortune*. Blink and you'd miss it.
```
Comment (cl8n) [inline news/2020-03-18-new-featured-artist-leaf.md]:
```suggestion
Tweeters @TheEggowo, @DrLou15, @iSpecYou and @sucuhaamane all take home the prize of correctly nabbing this week's hint. Congratulations! For those wondering, "verzweiflung" briefly features as accent text in the video for *Calamity Fortune*. Blink and you'd miss it.
```

File: news/2020-03-18-new-featured-artist-leaf.md (hunk 1 above)
Comment (cl8n) [reply news/2020-03-18-new-featured-artist-leaf.md]: rip pikaquim

Comment (Ephemeralis) [reply news/2020-03-18-new-featured-artist-leaf.md]: graveyard is the main issue as well

File: news/2020-03-18-new-featured-artist-leaf.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Check out this [Loved beatmap](https://osu.ppy.sh/beatmapsets/884669) by [Mr HeliX](https://osu.ppy.sh/users/2330619), and this [popular Loved osu!mania chart](https://osu.ppy.sh/beatmapsets/518951) by [jakads](https://osu.ppy.sh/users/259972)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/73/previews/1980.mp3" type="audio/mpeg">
+</audio>
+
+### Calamity Fortune
+
+Check out this [extremely popular classic beatmap](https://osu.ppy.sh/beatmapsets/65855) hosted by [Flower](https://osu.ppy.sh/users/1033017)!
```
Comment (Nivalyx) [inline news/2020-03-18-new-featured-artist-leaf.md]: Calamity Fortune's beatmap link is incorrect, should link to https://osu.ppy.sh/beatmapsets/96103 instead

**Pull request merged**